isa.c

```diff
@@ -78,9 +84,19 @@ static void decode_and_exec(uint32_t instruction)
             case OP_EOR: exec_EOR(instruction); break;
             case OP_ORR: exec_ORR(instruction); break;
             case OP_TST: exec_TST(instruction); break;
+            case OP_MOV: exec_MOV(instruction); break;
+            case OP_MVN: exec_MVN(instruction); break;
         }
     } else if (get_bits(instruction, 27, 24) == 0xf) { // SWI
         exec_SWI(instruction);
+    } else if (get_bits(instruction, 27, 24) == 0x0 && 
```

Comment:
yah so decoder will read these inside data processing instructions. if you want to fix it within this PR, reply "yay" else i'll do it later